doc/source/reference/distutils.rst

```diff
@@ -23,37 +23,25 @@ information is available in the :ref:`distutils-user-guide`.
 Modules in :mod:`numpy.distutils`
 =================================
 
-misc_util
----------
-
-.. module:: numpy.distutils.misc_util
+.. currentmodule:: numpy.distutils
 
 .. autosummary::
    :toctree: generated/
 
-   get_numpy_include_dirs
-   dict_append
-   appendpath
-   allpath
-   dot_join
-   generate_config_py
-   get_cmd
-   terminal_has_colors
-   red_text
-   green_text
-   yellow_text
-   blue_text
-   cyan_text
-   cyg2win32
-   all_strings
-   has_f_sources
-   has_cxx_sources
-   filter_sources
-   get_dependencies
-   is_local_src_dir
-   get_ext_source_files
-   get_script_files
+   ccompiler
+   cpuinfo.cpu
+   core.Extension
+   exec_command
+   log.set_verbosity
+   misc_util
+   system_info.get_info
+   system_info.get_standard_file
+
 
+Configuration class
+===================
+
+.. currentmodule:: numpy.distutils
 
 .. class:: Configuration(package_name=None, parent_name=None, top_path=None, package_path=None, **attrs)
```

Comment:
this class lives in `numpy.distutils.misc_util`

Reply:
Oh, thanks. somewhat thought there were two PRs failing.

Reply:
Why are you removing all those other entries?